scripts/buildVM-files/runQemuVM.sh

```diff
@@ -1,23 +1,45 @@
 #!/bin/bash
 set -e
 
-function usage() {
+# Populate the $macaddr variable with a randomly generated MAC address in the
+# 52:54:::: range.
+generate_random_mac() {
+	printf -v macaddr "52:54:%02x:%02x:%02x:%02x" \
+		$(( $RANDOM & 0xff )) $(( $RANDOM & 0xff )) \
```

Comment:
Do you need to seed RANDOM here before using it?

Reply:
The default seed for `$RANDOM` seems sufficient for what I'm using it for here. I stole it from a [helper script](https://github.com/ni/nilrt/blob/d77f1219b48ab71402c329eabd93e924679b2f64/scripts/tests/vpn-test-files/start_qemu_bridge.sh#L17) for the provisioning tests - which has a history of creating sufficiently random MAC addresses for its VMs.